namespaces/collectd.yml

```diff
@@ -0,0 +1,453 @@
+      description: >
+        collectd's type_instance contains type of the sapce: free/used/reserved.
+
+    - name: mounted_volume
+      type: string
+      description: >
+        collectd's plugin_instance contains the name of the mounted volume.
+
+    - name: df_complex
+      type: integer
+      description: >
+        collectd's type df_complex of plugin df.
+
+    - name: percent_bytes
+      type: float
+      description: >
+        collectd's type percent_bytes of plugin df.
+
+  - name: entropy
+    type: group
+    description: >
+      Corresponds to collectd's entropy plugin.
+    fields:
+
+    - name: entropy
+      type: integer
+      description: >
+        collectd's entropy type of entropy plugin.
+
+  - name: memory
+    type: group
+    description: >
+      Corresponds to collectd's memory plugin.
+    fields:
+
+    - name: memory_type
+      type: string
+      description: >
+        collectd's type_instance contains type of memory:
+        used/cached/free/slab_unrecl/slab_recl/buffered.
+        Additionally types available for virt plugin: actual_balloon/rss/total
+
+    - name: memory
+      type: integer
+      description: >
+        collectd's memory type of memory plugin.
+
+    - name: percent
+      type: float
+      description: >
+        collectd's percent type of memory plugin.
+
+  - name: swap
+    type: group
+    description: >
+      Corresponds to collectd's swap plugin.
+    fields:
+    - name: swap_type
+      type: string
+      description: >
+        collectd's type_instance contains type of swap:
+        used/cached/free.
+        and for type swap_io:
+        in/out
+
+    - name: swap
+      type: integer
+      description: >
+        collectd's swap type of swap plugin.
+
+    - name: swap_io
+      type: integer
+      description: >
+        collectd's swap_io type of swap plugin.
+
+  - name: load
+    type: group
+    description: >
+      Corresponds to collectd's load plugin.
+    fields:
+    - name: load
+      type: group
+      description: >
+        collectd's load type of load plugin
+      fields:
+      - name: shortterm
+        type: float
+        description: >
+          TODO
+
+      - name: midterm
+        type: float
+        description: >
+          TODO
+
+      - name: longterm
+        type: float
+        description: >
+          TODO
+
+  - name: aggregation
+    type: group
+    description: >
+      Corresponds to collectd's aggregation plugin.
+    fields:
+    - name: cpu_state
+      type: string
+      description: >
+        collectd's type_instance usually contains the name of the cpu state.
+        interrupt / user / wait / nice / softirq / system / idle / steal
+
+    - name: metric_type
+      type: string
+      description: >
+        type of aggregation metric: cpu-average/cpu-sum/used
+
+    - name: percent
+      type: float
+      description: >
+        TODO
+
+doc_sections:
+  - ["pod_log", "Openshift logging metadata"]
```

Comment:
I think this is incorrect.
The default fields from Collectd are:
plugin, type, type_instance, Plugin_instance, dstypes, dsnames, values

We added the fields hostname, entity.
We also need to update the values field since some of the plugins report a number and some report an array of numbers.